layouts/parts/history.erb

```diff
@@ -0,0 +1,218 @@
+          <div>
+            <time>4 oktober 2001</time> Hoger Instituut voor Lich. Opv. en Kinesitherapie (HILOK)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>23 oktober 2002</time> Hoger Instituut voor Lich. Opv. en Kinesitherapie (HILOK)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>22 oktober 2003</time> ???
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>27 oktober 2004</time> Hoger Instituut voor Lich. Opv. en Kinesitherapie (HILOK)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>1 maart 2006 (?)</time> ???
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>28 februari 2007</time> Hoger Instituut voor Lich. Opv. en Kinesitherapie (HILOK)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>9 april 2008</time> Vlaamse Technische Kring (VTK)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>4 maart 2009</time> Vlaamse Technische Kring (VTK)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>21 april 2010</time> Vlaamse Technische Kring (VTK)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>4 mei 2011</time> Hoger Instituut voor Lich. Opv. en Kinesitherapie (HILOK)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>25 april 2012</time> Hoger Instituut voor Lich. Opv. en Kinesitherapie (HILOK)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>24 april 2013</time> Vlaamse Technische Kring (VTK)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>30 april 2014</time> Vlaamse Technische Kring (VTK)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>29 april 2015</time> Vlaamse Technische Kring (VTK)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>15-16</time> ???
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>26 april 2017</time> Hoger Instituut voor Lich. Opv. en Kinesitherapie (HILOK)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>26 april 2018</time> Vlaamse Technische Kring (VTK)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>29 april 2019</time> Vlaamse Technische Kring (VTK)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>29 april 2020</time> (niet doorgegaan door COVID-19)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>26-29 april 2021</time> (niet doorgegaan door COVID-19)
+          </div>
+        </li>
+        <li>
+          <div>
+            <time>27 april 2022</time> Hoger Instituut voor Lich. Opv. en Kinesitherapie (HILOK)
+          </div>
+        </li>
+        <li></li>
+      </ol>
```

Comment:
Zou ge deze waslijst in een file onder de data folder kunnen steken? en de lijst gewoon vervangen me een for loop? Dan is da meer in lijn moet hoe andere data opgeslagen wordt op de website

Reply:
Good point, done @NuttyShrimp